addons/website_event/views/event_templates_list.xml

```diff
@@ -160,51 +137,127 @@
 <!-- Filter - Date -->
 <template id="event_time" inherit_id="website_event.index_topbar" name="Filter by Date">
     <xpath expr="//div[hasclass('o_wevent_search')]" position="before">
-        <div class="dropdown d-none d-lg-block me-2 my-1">
-            <a href="#" role="button" class="btn btn-light dropdown-toggle" data-bs-toggle="dropdown">
+        <div class="dropdown d-none d-lg-block">
+            <a href="#" role="button" class="btn btn-light dropdown-toggle" data-bs-toggle="dropdown" title="Filter by Date">
```

Comment:
May need to add this in the .pot

Not sure these changes are relevant so maybe omit them? Or have them in a separate commit at least.
(i.e. don't fixup and keep a separate commit with a description of the changes)